content/plugins/environment-plugin.md

```diff
@@ -1,24 +1,62 @@
 ---
 title: environment-plugin
 contributors:
-  - simon04
+  - einarlove
```

Comment:
😨  Sorry @simon04, It was not intentional. I merged this obviously to quick without much thought.